avocado/core/job.py

```diff
@@ -377,6 +377,8 @@ def _log_avocado_datadir():
         job_log.info('')
```

Comment:
For now it won't be used, but eventually the avocado-virt should use it. I'll add selftests for now...